plugins/obs-webrtc/obs-webrtc.cpp

```diff
@@ -0,0 +1,19 @@
+#include <obs-module.h>
+
+#include "whip-output.h"
+#include "whip-service.h"
+
+OBS_DECLARE_MODULE()
+OBS_MODULE_USE_DEFAULT_LOCALE("obs-webrtc", "en-US")
+MODULE_EXPORT const char *obs_module_description(void)
+{
+	return obs_module_text("Module.Description");
```

Comment:
```suggestion
	return obs_module_text("OBS WHIP output & service");
```
Doesn't have to be localized; see rtmp & other outputs for instance.